cpu/cpu_test.go

```diff
@@ -160,7 +158,7 @@ func (cis *CPUInfoSuite) TestGetMetricTypes() {
 
 				namespaces := []string{}
 				for _, m := range mts {
-					namespaces = append(namespaces, strings.Join(m.Namespace().Strings(), "/"))
+					namespaces = append(namespaces, strings.Join(m.Namespace.Strings(), "/"))
```

Comment:
You need to add  additional`/` in expected namespace. It means that you need to change  `'intel/procfs/cpu/*/user_percentage'` to `'/intel/procfs/cpu/*/user_percentage'`.